diagnostics-extension/tsconfig.json

```diff
@@ -11,10 +11,11 @@
         "../common/*"
       ]
     },
+    "moduleResolution": "node",
+    "noImplicitAny": false,
```

Comment:
needed for importing cros-dpsl-js library.

> "moduleResolution": "node",

[moduleResolution](https://www.typescriptlang.org/tsconfig#moduleResolution) is needed since dpsl lib is using CommonJS syntax which is the standard for Node.js. The default is classic, which is used for older versions of Typescript (and doesn't work with dpsl right now).

> "noImplicitAny": false,

dpsl object from cros-dpsl-js has an implicit type Any, the default for this field is true, which prevents the usage of implicit any. The proper handling of this is to define a declaration file (.d.ts) which defines the type of the imported objects, but I am not sure if we want to mix .ts with .js files in dpsl.